Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -155,15 +155,15 @@ open class AVFoundationPlayback: Playback {
 
     open override var position: Double {
         if isDvrAvailable, let start = dvrWindowStart,
-            let position = player?.currentItem?.currentTime().seconds {
```

Comment:
Maybe we could rename this to make it less confusing? It's a bit misleading cause the currentTime is not the position itself (we still need to subtract the DVR window start).

Reply:
Done in 7c88573060002f47bf0976706b4c592bfa0b8866